src/account/AccountStorage.sol

```diff
@@ -5,7 +5,7 @@ import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet
 
 import {HookConfig, ModuleEntity} from "../interfaces/IModularAccount.sol";
 
-// bytes = keccak256("ERC6900.UpgradeableModularAccount.Storage")
+// bytes = keccak256("ERC6900.ReferenceModularAccount.Storage")
 bytes32 constant _ACCOUNT_STORAGE_SLOT = 0x9f09680beaa4e5c9f38841db2460c401499164f368baef687948c315d9073e40;
```

Comment:
This storage slot should change since the hash changed

Reply:
ahh good catch, updating...